core/approval/errors.go

```diff
@@ -1,35 +0,0 @@
-package approval
-
-import "errors"
-
-var (
-	ErrPolicyNotFound                 = errors.New("policy not found")
-	ErrDependencyApprovalStepNotFound = errors.New("unable to resolve approval step dependency")
-	ErrApprovalStepConditionNotFound  = errors.New("unable to resolve designated condition")
-	ErrNilResourceInAppeal            = errors.New("unable to resolve resource from the appeal")
-	ErrInvalidConditionField          = errors.New("invalid condition field")
-
-	ErrAppealStatusCanceled           = errors.New("appeal already canceled")
-	ErrAppealStatusApproved           = errors.New("appeal already approved")
-	ErrAppealStatusRejected           = errors.New("appeal already rejected")
-	ErrAppealStatusBlocked            = errors.New("approval is blocked")
-	ErrAppealStatusUnrecognized       = errors.New("unrecognized appeal status")
-	ErrAppealDuplicate                = errors.New("appeal with the same resource and role already exists")
-	ErrAppealInvalidExtensionDuration = errors.New("invalid appeal extension duration")
-	ErrAppealFoundActiveGrant         = errors.New("user still have an active grant")
-	ErrGrantNotEligibleForExtension   = errors.New("existing grant is not eligible for extension")
-	ErrCannotCreateAppealForOtherUser = errors.New("creating appeal for other individual user (account_type=\"user\") is not allowed")
-
-	ErrApprovalDependencyIsBlocked = errors.New("found previous approval step that is still in blocked")
-	ErrApprovalDependencyIsPending = errors.New("found previous approval step that is still in pending")
-	ErrApprovalStatusApproved      = errors.New("approval already approved")
-	ErrApprovalStatusRejected      = errors.New("approval already rejected")
```

Comment:
All of these errors in this file are not required?

Reply:
yes, looks like this is copied from `core/appeal/errors.go` but not sure why it was copied